### ISSUE TITLE
[Bug Report][3.1.2] Accessibility issue on form components

### ISSUE THREAD

**Issue by Chris-McQuiggan** [vuetifyjs/vuetify#16554]
### Environment
**Vuetify Version:** 3.1.2
**Vue Version:** 3.2.45
**Browsers:** Chrome 109.0.0.0
**OS:** Windows 10, Mac OSX

### Steps to reproduce
Steps:
1. Install axe dev tools
2. Go to reproduction link
3. Run axe dev tools scan
4. OBSERVE issues on the form components

### Expected Behavior
All accessibility tests to pass

### Actual Behavior
2 categories of failures on the form components
  - ARIA input fields must have accessible name
  - Elements must only use allowed ARIA attributes

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqVU01z2yAQ/SuUi9sZSzhJc/EonfTWf9CD8QFLa5cGAQPITcbj/94FyRnwRzu5afc93nssq9WBfre23g9Al7QJ0FslAnzjmpBmXwlr02cqeiH1VKU6wGuothJURxA0HagnTnu/45QosRmrNxJZJLGwL5wU1S2Q5eKt6TdmY14vpJcSQ3oslfShtDqduWKUQYWNBwVt+IjJdOK6TQEWRmIIBkNYBQE+Ypefu2J4BheWcbjCgfjv60TSrceZsHJoSnbgMtm9UEO0zyeRONdGNAGFohN6B5e6qV3qptZt9RI+eTQs295YjHvdsGzfsfStkzbgI4YhwrK3xgVyIA625Ei2zvRkhj/KLJJbo30gOE/yFPHPsx+glCE/jVPdp9mXEyFNZqI8LN7bY8yxvbpbzMnjYv0OxiU4YZwmXU7nhNMkjp/oWifheKZhY2wMTOd0zIyXtfVvbzT+04d4Zz4BntMlSZ3Yw7vEmtNfIVi/ZGzQ9mVX4z6xZ8SYG3SQPVSd6Z8f6vv66yPrMFver8H31caZPx4cOmK4TDzI7dstgwlG3bv6fpSdWlHyQoqh/h5c5UDjy8a3/UfuM26R/Qy7yB89j1wfcZZK6hd/NsbWpxGu1pFynNNRGsnpHnT9F5RGxKs=)

### Other comments
**"ARIA input fields must have accessible name"**
This problem seems to be caused by a role of "textbox" being defined on a parent div and the defined aria label being attached to the input.
***Affected components:***
- v-text-field
- v-combobox
- v-select
- v-autocomplete
- v-textarea
- v-slider
- v-range-slider
- v-color-picker
- v-data-table

**"Elements must only use allowed ARIA attributes"**
Caused by aria-expanded being defined on unexpected element's role
***Affected components:***
- v-combobox
- v-select
- v-autocomplete
- v-expansion-panel

**Other failures of note**
- "Buttons must have discernible text"
  - v-color-picker
  - v-data-table
- "Form elements must have labels"
  - v-color-picker
  - v-data-table

**Comment by Harm-Nullix** [comment]
I find there are some issues we cannot resolve ourselves using Vuetify, especially with forms and dialog/menu's.

I find myself not being able to inject all the needed attributed and orders into the components without rewriting them myself, which is a bummer. 

How can I help in a efficient way?

@KaelWD , @johnleider , I do not really know who to target for these things, but is there an easy way to move in and help on these accessibility problems? Accessibility is a major +1 on using Vuetify as a framework if implemented correctly and I find it should pass the wcag (https://www.w3.org/TR/WCAG21/) from the Axe core with a breeze.

**Comment by johnleider** [comment]
> I find there are some issues we cannot resolve ourselves using Vuetify, especially with forms and dialog/menu's.
> 
> I find myself not being able to inject all the needed attributed and orders into the components without rewriting them myself, which is a bummer.
> 
> How can I help in a efficient way?
> 
> @KaelWD , @johnleider , I do not really know who to target for these things, but is there an easy way to move in and help on these accessibility problems? Accessibility is a major +1 on using Vuetify as a framework if implemented correctly and I find it should pass the wcag ([w3.org/TR/WCAG21](https://www.w3.org/TR/WCAG21/)) from the Axe core with a breeze.

Reach out to me on [Discord](https://community.vuetifyjs.com)

**Comment by djurdjen** [comment]
Any news on whether  this is being looked at or not? I'm also finding myself in the situation where i'm forced to rewrite the component myself without vuetify because of the AXE tests output. Which is a shame because Vuetify offers a lot of great functionality I would like to keep using!

**Comment by johnleider** [comment]
> Any news on whether this is being looked at or not? I'm also finding myself in the situation where i'm forced to rewrite the component myself without vuetify because of the AXE tests output. Which is a shame because Vuetify offers a lot of great functionality I would like to keep using!
> 
> If there's any way I can help, let me know!

Reach out to me on [Discord](https://community.vuetifyjs.com/)
